scripts/start/readCheckScenarioConfig.R

```diff
@@ -59,6 +59,15 @@ readCheckScenarioConfig <- function(filename, remindPath = ".", testmode = FALSE
       whitespaceErrors <- whitespaceErrors + sum(haswhitespace)
     }
   }
+  missingRealizations <- 0
+  modules <- gms::getModules(file.path(remindPath, "modules"))
+  for (m in intersect(rownames(modules), colnames(scenConf))) {
+    missingRealiz <- setdiff(unique(scenConf[, m]), c(NA, strsplit(modules[m, "realizations"], ",")[[1]]))
+    if (length(missingRealiz) > 0) {
+      warning("For module ", m, ", these realizations are missing: ", paste0(missingRealiz, collapse = ", "))
```

Comment:
You might want to add the list of scenarios that select a missing realization.
```suggestion
            warning("For module ", m, ", these realizations ", paste0(missingRealiz, collapse = ", "),
              " are missing although selected in ", paste(rownames(scenConf)[scenConf[,m] %in% missingRealiz], collapse = ", "))

```